3.10/aql/functions-geo.md

```diff
@@ -47,12 +47,29 @@ FOR doc IN doc // e.g. documents returned by a traversal
 Checks whether the [GeoJSON object](../indexing-geo.html#geojson) `geoJsonA`
 fully contains `geoJsonB` (Every point in B is also in A). The object `geoJsonA` has to be of type 
 `Polygon` or `MultiPolygon`, other types are not supported because containment is ill defined. 
-This function can be **optimized** by a S2 based [geospatial index](../indexing-geo.html).
 
 - **geoJsonA** (object): first GeoJSON object or coordinate array (in longitude, latitude order)
 - **geoJsonB** (object): second GeoJSON object or coordinate array (in longitude, latitude order)
 - returns **bool** (bool): true when every point in B is also contained in A, false otherwise
 
+A query containing a FILTER expression of the form
+
+```
+  FOR doc IN collectionname
+    FILTER GEO_CONTAINS(geoJson, doc.geo)
+    ...
+```
+
+can be **optimized** by an S2 based [geospatial index](../indexing-geo.html) 
+on the attribute `geo` of the collection `collectionname`, if `geoJson`
+evaluates to a valid `geojson` object, and the index has the `geoJson`
+flag set to `true`. Note that it has to be in this direction,
```

Comment:
@neunhoef what do you mean by `direction`?

Reply:
I meant "order of arguments", clarified.